integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -172,6 +173,13 @@ case class CreateTable(cm: TableModel) extends RunnableCommand {
 
     val tableInfo: TableInfo = TableNewProcessor(cm)
 
+    // Add validation for sort scope when create table
+    val sortScope = tableInfo.getFactTable.getTableProperties.get("sort_scope")
+    if (null != sortScope && !CarbonUtil.isValidSortOption(sortScope)) {
+      throw new InvalidConfigurationException("The sort scope " + sortScope
```

Comment:
suggest to change to 
```
s"Passing invalid sort scope '$sortScope', valid sort scopes are 'NO_SORT', 'BATCH_SORT', 'LOCAL_SORT' or 'GLOBAL_SORT' "
```

Reply:
For this, I just keep same with error message which is already exists.